From 4a07f7933aad6b3f3e533fa69e5401d82415b319 Mon Sep 17 00:00:00 2001
From: Reiner Herrmann <reiner@reiner-h.de>
Date: Mon, 30 Jan 2017 21:27:16 +0100
Subject: Fix detection of libcurl

curl headers are now in a multiarch include directory, so the current check no
longer succeeds. Check for curl-config instead, which is provided by the same
-dev package as the headers.

Bug-Debian: https://bugs.debian.org/852894
Bug-Upstream: https://bugzilla.clamav.net/show_bug.cgi?id=11739
Patch-Name: Fix_detection_of_libcurl.patch
---
 m4/reorganization/libs/curl.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/reorganization/libs/curl.m4 b/m4/reorganization/libs/curl.m4
index 6f29cbf..5d1fd01 100644
--- a/m4/reorganization/libs/curl.m4
+++ b/m4/reorganization/libs/curl.m4
@@ -18,7 +18,7 @@ fi
 AC_MSG_RESULT([$LIBCURL_HOME])
 ])
 
-if test ! -f "$LIBCURL_HOME/include/curl/curl.h"
+if test ! -f "$LIBCURL_HOME/bin/curl-config"
 then
     AC_MSG_WARN([libcurl not found. Please use the web interface for submitting FPs/FNs.])
 else
